src/content/docs/apm/agents/nodejs-agent/installation-configuration/codestream-integration.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: New Relic CodeStream integration for Node.js
+tags:
+  - Agents
+  - Nodejs agent
+  - Configuration
+metaDescription: The Node.js agent can be integrated with New Relic CodeStream to provide code level metrics.
+---
+
+Display context-sensitive APM data directly in your IDE by integrating [New Relic CodeStream](/docs/codestream/how-use-codestream/performance-monitoring#code-level) with the Node.js agent. Visualize code-level production telemetry in your editor as your write and review code with this integration.
+
+## Get started [#get-started]
+
+First, [install](/docs/codestream/start-here/install-codestream) the New Relic CodeStream extension into your supported IDE of choice, and log in.
```

Comment:
Just to confirm, they're logging in _before_ they install, not _after_?

Reply:
frankly this is copy/pasted from an already approved doc for the .net agent https://docs.newrelic.com/docs/apm/agents/net-agent/other-features/net-codestream-integration/